src/pages/JobSubmitted.tsx

```diff
@@ -25,17 +22,17 @@ export default function JobSubmitted() {
   const [jobNotes, setJobNotes] = useState('');
   const [workflows, setWorkflows] = useState<Workflow[]>([]);
 
-  const { loading, error, data } = useQuery(GET_JOB_BY_ID, {
+  const { loading, error, data } = useQuery(GET_JOB_BY_ID, { //should later check logged in user token to access
     variables: { id: jobId },
+    skip: !jobId,
```

Comment:
Just like [the other jobId check was unnecessary](https://github.com/hicsail/damplab-ui/pull/46#discussion_r2274144447), this one is too, because the router doesn't let you get to this page if there's no jobId in the url. Try it: you can check for `!jobId` somewhere in this component and add a log, and then navigate to `/jobs` or `/jobs/` and see what happens